storage/test/authz_models.go

```diff
@@ -15,48 +15,40 @@ import (
 
 func WriteAndReadAuthorizationModelTest(t *testing.T, datastore storage.OpenFGADatastore) {
 	ctx := context.Background()
-	store := id.Must(id.New()).String()
+	storeID := id.Must(id.New()).String()
 
-	model := &openfgapb.AuthorizationModel{
-		Id:            id.Must(id.New()).String(),
-		SchemaVersion: "1.0",
-		TypeDefinitions: []*openfgapb.TypeDefinition{
-			{
-				Type: "folder",
-				Relations: map[string]*openfgapb.Userset{
-					"viewer": {
-						Userset: &openfgapb.Userset_This{
-							This: &openfgapb.DirectUserset{},
-						},
-					},
-				},
-			},
-		},
-	}
+	t.Run("write, then read, succeeds", func(t *testing.T) {
+		model := &openfgapb.AuthorizationModel{
+			Id:              id.Must(id.New()).String(),
+			SchemaVersion:   "1.0",
+			TypeDefinitions: []*openfgapb.TypeDefinition{{Type: "folder"}},
+		}
 
-	err := datastore.WriteAuthorizationModel(ctx, store, model)
-	require.NoError(t, err)
+		err := datastore.WriteAuthorizationModel(ctx, storeID, model)
+		require.NoError(t, err)
 
-	got, err := datastore.ReadAuthorizationModel(ctx, store, model.Id)
-	require.NoError(t, err)
+		got, err := datastore.ReadAuthorizationModel(ctx, storeID, model.Id)
+		require.NoError(t, err)
 
-	cmpOpts := []cmp.Option{
-		cmpopts.IgnoreUnexported(
-			openfgapb.AuthorizationModel{},
-			openfgapb.TypeDefinition{},
-			openfgapb.Userset{},
-			openfgapb.Userset_This{},
-			openfgapb.DirectUserset{},
-		),
-	}
+		cmpOpts := []cmp.Option{
```

Comment:
We use these same compare options in multiple places. Can we just define this as a global in the `test/storage.go` . We can add every type that we want the unexported fields to ignore to the list and reuse it everywhere we compare protobuf structs.

Reply:
Oh yeah. I didn't see that. Updated. Thanks!